datahub-web-react/src/app/entity/user/UserInfoSideBar.tsx

```diff
@@ -0,0 +1,347 @@
+    line-height: 22px;
+    color: #595959;
+    margin-bottom: 7px;
+`;
+
+export const Team = styled.div`
+    font-size: 12px;
+    line-height: 20px;
+    color: #8c8c8c;
+`;
+
+export const SocialDetails = styled.div`
+    font-size: 12px;
+    line-height: 20px;
+    color: #262626;
+    text-align: left;
+    margin: 6px 0;
+`;
+
+export const EditProfileButton = styled.div`
+    bottom: 24px;
+    position: absolute;
+    right: 27px;
+    width: 80%;
+    left: 50%;
+    -webkit-transform: translateX(-50%);
+    -moz-transform: translateX(-50%);
+    transform: translateX(-50%);
+
+    button {
+        width: 100%;
+        font-size: 12px;
+        line-height: 20px;
+        color: #262626;
+    }
+`;
+
+export const AboutSection = styled.div`
+    text-align: left;
+    font-weight: bold;
+    font-size: 14px;
+    line-height: 22px;
+    color: #262626;
+`;
+
+export const AboutSectionText = styled.div`
+    font-size: 12px;
+    font-weight: 100;
+    line-height: 15px;
+    padding: 5px 0;
+
+    &&& .ant-typography {
+        margin-bottom: 0;
+    }
+    &&& .ant-typography-edit-content {
+        padding-left: 15px;
+        padding-top: 5px;
+    }
+`;
+
+export const GroupsSection = styled.div`
+    text-align: left;
+    font-weight: bold;
+    font-size: 14px;
+    line-height: 22px;
+    color: #262626;
+`;
+
+export const TagsSection = styled.div`
+    height: calc(75vh - 460px);
+    padding: 5px;
+    // overflow: auto;
+`;
+
+export const NoDataFound = styled.span`
+    font-size: 12px;
+    color: #262626;
+    font-weight: 100;
+`;
+
+export const Tags = styled.div`
+    margin-top: 5px;
+`;
+
+export const GroupsSeeMoreText = styled.span`
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 20px;
+    color: #1890ff;
+    cursor: pointer;
+`;
+
+/**
+ * Responsible for reading & writing users.
+ */
+export default function UserInfoSideBar() {
+    const { urn: encodedUrn } = useUserParams();
+    const urn = decodeUrn(encodedUrn);
+    const [updateCorpUserPropertiesMutation] = useUpdateCorpUserPropertiesMutation();
+    const { loading, error, data, refetch } = useGetUserQuery({ variables: { urn, groupsCount: GROUP_PAGE_SIZE } });
+    const entityRegistry = useEntityRegistry();
+
+    const [groupSectionExpanded, setGroupSectionExpanded] = useState(false);
+    const [editProfileModal, showEditProfileModal] = useState(false);
+    /* eslint-disable @typescript-eslint/no-unused-vars */
+    const [editableAboutMeText, setEditableAboutMeText] = useState<string | undefined>('');
+
+    const groupsDetails = data?.corpUser?.relationships as ExtendedEntityRelationshipsResult;
+    const photoUrl = data?.corpUser?.editableProperties?.pictureLink || undefined;
+
+    if (error || (!loading && !error && !data)) {
+        return <Alert type="error" message={error?.message || 'Profile data failed to load'} />;
+    }
+
+    // EditProfile modal Constants
+    const getEditModalData = () => {
+        return {
```

Comment:
Why can't this be a const variable?

const modalData = {
   urn: urn,
   .... 
}

Reply:
new changes are added